internal/clients/account_identity.go

```diff
@@ -0,0 +1,9 @@
+package clients
+
+type AccountIdentity struct {
+	AWSDetails *AccountDetailsAWS `json:"aws,omitempty" yaml:"aws"`
+}
+
+type AccountDetailsAWS struct {
+	AccountID string `json:"account_id" yaml:"account_id"`
+}
```

Comment:
You could also just put string directly into the payload type, we use embedding mainly for models. There is no need for intermediate type, but it is fine.

Reply:
I wasn't sure, since this object might contains Details for azure and stuff in future too, I thought it might be useful to be able to add fields to the details at some point too.